tiledb/tests/cc/test_cc.py

```diff
@@ -160,6 +161,39 @@ def test_array():
     arr.close()
 
 
+@pytest.mark.xfail
+def test_consolidate_fragments():
+    uri = tempfile.mkdtemp()
+    ctx = lt.Context()
+    config = lt.Config()
+
+    tiledb.from_numpy(uri, np.random.rand(4)).close()
+
+    with tiledb.open(uri, "w") as A:
+        A[:] = np.random.rand(4)
+
+    with tiledb.open(uri, "w") as A:
+        A[:] = np.random.rand(4)
+
+    schema = tiledb.ArraySchema.load(uri, ctx=ctx)
+    fragment_info = PyFragmentInfo(uri, schema, False, ctx)
+
+    assert fragment_info.get_num_fragments() == 3
+
+    uris = fragment_info.get_uri()
+    # get fragment name form alone, not the full path(s) (the part of each uri after the last /)
+    # https://github.com/TileDB-Inc/TileDB-Py/pull/1946
+    uris = [uri.split("/")[-1] for uri in uris]
+
+    arr = lt.Array(ctx, uri, lt.QueryType.WRITE)
+    arr.consolidate_fragments(ctx, uris, config)
+    arr.close()
+
+    fragment_info = PyFragmentInfo(uri, schema, False, ctx)
+    # This xfail will be resolved after SC-10782
+    assert fragment_info.get_num_fragments() == 4
```

Comment:
Since it is expected right now, let's remove the x-fail, set `assert .. == 1`, and just make a note that fragmentinfo doesn't see the consolidated range.